.github/workflows/lint.yml

```diff
@@ -87,3 +87,37 @@ jobs:
         uses: hiwelo/new-dependencies-action@1.0.1
         with:
           token: ${{ secrets.OKP4_TOKEN }}
+
+  lint-branch-name:
+    runs-on: ubuntu-20.04
+    if: github.actor != 'dependabot[bot]' && github.event_name == 'pull_request' && (github.event.action == 'opened' || github.event.action == 'reopened')
+    steps:
+      - name: Check branch name conventions
+        uses: AlbertHernandez/branch-name-action@v1.0.2
+        with:
+          branch_pattern: "feat|fix|hotfix|docs|ci|chore"
```

Comment:
That sounds great!
What about 'style' and 'build' pull requests?

Reply:
Yes I'll add it.

Reply:
Updated the description as well (by https://github.com/okp4/ui/pull/193/commits/e9db80ea1cc751e97f0ff8eb24d0ab622aa7f4d3).